rtl/src/errors.rs

```diff
@@ -18,6 +18,9 @@ pub enum Error {
     GCore(#[from] GCoreError),
     #[error("codec: {0}")]
     Codec(#[from] CodecError),
+    #[cfg(not(target_arch = "wasm32"))]
```

Comment:
Can we go w/o conditional compilation here?

Reply:
No, gclient does not compile for wasm32